src/helper.h

```diff
@@ -32,6 +32,8 @@
 
 class Packable;
 class wordlist;
+class MemBuf;
+class CommTimeoutCbParams;
```

Comment:
alphabetical list please.
```suggestion
class CommTimeoutCbParams;
class MemBuf;
class Packable;
class wordlist;
```